ceno_zkvm/src/tables/range/range_impl.rs

```diff
@@ -172,3 +172,98 @@ impl DynamicRangeTableConfig {
         Ok([witness, structural_witness])
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct DoubleRangeTableConfig {
+    range_a: StructuralWitIn,
+    range_b: StructuralWitIn,
+    mlt: WitIn,
+}
+
+impl DoubleRangeTableConfig {
+    pub fn construct_circuit<E: ExtensionField>(
+        cb: &mut CircuitBuilder<E>,
+        rom_type: ROMType,
+        range_a_bits: usize,
+        range_b_bits: usize,
+    ) -> Result<Self, CircuitBuilderError> {
```

Comment:
we should keep `range_a_bits`, `range_b_bits` bit in `DoubleRangeTableConfig` so in `assign_instance` we can retrieve from config and avoid potiential mismatched

Reply:
fixed in https://github.com/scroll-tech/ceno/pull/1044/commits/ef35883ecf4fe51575d9434f5969e2cfe98a2741 :)